OpenMarket/OpenMarket/Utils/RequestAssistant.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  RequestAssistant.swift
+//  OpenMarket
+//
+//  Created by Grumpy, OneTool on 2022/05/10.
+//
+
+import Foundation
+
+
+final class RequestAssistant {
+    private let sessionManager = URLSessionGenerator(session: URLSession.shared)
+    
+    func requestListAPI(pageNum: Int, items_per_page: Int, completionHandler: @escaping ((Result<ProductList, OpenMarketError>) -> Void)) {
```

Comment:
변수, 파라미터 네이밍은 camelCase 준수하는 것을 권장드립니다.
```suggestion
    func requestListAPI(pageNum: Int, itemsPerPage: Int, completionHandler: @escaping ((Result<ProductList, OpenMarketError>) -> Void)) {
```